lib/platform/gitlab/__snapshots__/index.spec.ts.snap

```diff
@@ -2039,6 +2039,61 @@ Array [
 ]
 `;
 
+exports[`platform/gitlab/index initRepo should fall back respecting when GITLAB_IGNORE_REPO_URL is set 1`] = `
+Array [
+  Array [
+    Object {
+      "cloneSubmodules": undefined,
+      "defaultBranch": "master",
+      "gitAuthorEmail": undefined,
+      "gitAuthorName": undefined,
+      "ignorePrAuthor": undefined,
+      "mergeMethod": "merge",
+      "repository": "some%2Frepo%2Fproject",
+      "url": undefined,
```

Comment:
You test seems to behave wrong, this url shouldn't be `undefined`

Reply:
sure, use `URL.format`

Reply:
Done (also tested on our gitlab)